website/docs/docs/dbt-cloud-apis/migrating-to-v2.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Migrating to dbt Cloud Administrative API v2"
+description: "You should migrate to API v2 while we deprecate API v4 "
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own:
+
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
```

Comment:
One minor nit, do we have precedent yet for handling URLs where we need to reference emea/au/us separately?

Reply:
I've addressed this! Thanks for bringing it up.